units/crons.go

```diff
@@ -113,20 +113,6 @@ func PopulateRepotrackerPollingJobs(part int) amboy.QueueOperation {
 
 func PopulateActivationJobs(part int) amboy.QueueOperation {
 	return func(queue amboy.Queue) error {
-		flags, err := evergreen.GetServiceFlags()
-		if err != nil {
-			return errors.WithStack(err)
-		}
-
-		if flags.TaskDispatchDisabled {
-			grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
-				"message": "task dispatching disabled",
-				"mode":    "degraded",
-				"impact":  "skipping stepback activation",
-			})
-			return nil
-		}
-
```

Comment:
why did you remove this?

Reply:
I suggested it. I don't think there's a reason we need to degrade this, as it merely activates versions whose batch times have passed. It doesn't change the size of the queue, and if it were not to run (as part of the scheduler being degraded), it would simply run once we undegraded the scheduler. Since it doesn't put significant load on the system and doesn't affect the queues, I don't think there's a reason to include it in a degraded flag.

Reply:
it can increase the number of tasks that enter the task queues. 

It's weird that it's here and not in the repotracker flag, but I don't think that we should exempt it from degraded mode.

Reply:
> it can increase the number of tasks that enter the task queues.

Sort of. Whether or not the scheduler is degraded determines whether or not tasks enter task queues, and degrading the scheduler stops new tasks from entering the queue altogether. Ultimately the effect of degrading this is merely whether boxes change color from light to dark gray in the UI. We can control the effect of this with other flags, which have clearer effects. That this is part of the repotracker is an historical accident. Batch time isn't logically part of tracking repositories.

Reply:
after a harrowing debate, we agreed that it should be behind the scheduler's flag.

Reply:
@ybrill - The principle we settled on is that more jobs behind degraded flags is better. It doesn't logically fit in the repotracker, since it doens't have to do with talking to github, but it is an example of changing state in a way that ultimately adds things to queues, so the scheduler works here.

Reply:
Done.